tests/PHPStan/Analyser/data/if.php

```diff
@@ -218,6 +218,7 @@ function () {
 
 	$nullOverwrittenInSwitchToOne = null;
 	switch (doFoo()) {
+		case 0:
```

Comment:
had to add this case, as otherwise phpstan detects after this PR, that the Exception would always be thrown and `$nullOverwrittenInSwitchToOne` can never be reached